core/src/main/scala/chisel3/internal/Builder.scala

```diff
@@ -363,8 +372,23 @@ private[chisel3] class ChiselContext() {
   val viewNamespace = Namespace.empty
 }
 
+/** Stores a [[Definition]] that is imported so that its Instances can be
+  * compiled separately.
+  */
+case class ImportedDefinitionAnnotation[T <: BaseModule with IsInstantiable](importedDefinition: Definition[T])
```

Comment:
There must be a better place to put this class than in `Builder.scala`. Is there another place that annotations are defined, within the `core` package?

Reply:
Would it make sense to stick in `Definition.scala`?

Reply:
Yeah, I think that's a good place.